src/backend/index.ts

```diff
@@ -11,17 +13,29 @@ import descriptionBulletsRouter from './src/routes/description-bullets.routes';
 const app = express();
 const port = process.env.PORT || 3001;
 
+// cors options
+const allowedHeaders = process.env.NODE_ENV === 'production' ? prodHeaders : '*';
 const options: cors.CorsOptions = {
-  origin: '*',
-  methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
```

Comment:
Just wondering why we only use GETs and POSTs, feel free to put this in slack if you feel this discussion belongs there.

Reply:
as of now we only have get and post requests so i'd figure i'd just limit it to that. we have a delete work package/project epic in the works so we'll probably add some delete endpoints for that. also i know that technically put and post have distinct meanings and stuff but i dont really see the point when they do the same thing ngl

Reply:
what does taking them out of the methods here do?

Reply:
You will only be allowed to make POST or DELETE calls, none of the rest anymore.

Reply:
"Taking them out" restricts someone from making the other types of HTTP requests.

Reply:
Taking them out of methods will prevent someone from performing those types of requests on the backend. I.e. you will only be allowed to POST or DELETE